doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1856 @@
+    - [3.3.5 Fdbsyncd](#335-Fdbsyncd)
+    - [3.3.6 Vxlanorch](#336-Vxlanorch)
+    - [3.3.7 Fdborch](#337-Fdborch)
+    - [3.3.8 L2nhgorch](#338-L2nhgorch)
+    - [3.3.9 EvpnMhOrch](#339-EvpnMhOrch)
+    - [3.3.10 ShlOrch](#3310-ShlOrch)
+  - [3.4 SAI](#34-SAI)
+    - [3.4.1 New SAI Objects](#341-new-sai-objects)
+    - [3.4.2 Changes to Existing SAI Objects](#342-changes-to-existing-sai-objects)
+    - [3.4.3 SAI Object Usage](#343-sai-object-usage)
+  - [3.5 CLI](#35-CLI)
+    - [3.5.1 Configuration Commands](#351-Configuration-Commands)
+    - [3.5.2 Show Commands](#352-Show-Commands)
+    - [3.5.3 Debug Commands](#353-Debug-Commands)
+    - [3.5.4 REST API Support](#354-REST-API-Support)
+- [4 Flow Diagrams](#4-Flow-Diagrams)
+- [5 Error Handling](#5-Error-Handling)
+- [6 Serviceability and Debug](#6-Serviceability-and-Debug)
+- [7 Warm Boot Support](#7-Warm-Boot-Support)
+- [8 Scalability](#8-Scalability)
+- [9 Unit Test](#9-Unit-Test)
+  - [9.1 Functional Test Cases](#91-Functional-Test-Cases)
+
+
+# List of Tables
+
+[Table 1: Abbreviations](#Table-1-abbreviations)
+
+# Revision
+
+| Rev  | Date       | Authors                                          | Change Description |
+| ---- | ---------  | ------------------------------------------------ | ------------------ |
+| 0.x  | Feb-2024   | Mike Mallin, Patrice Brissette, Naveen Gamini  | Cisco initial draft  |
+| 0.x  | Feb-2024   | Syed Hasan Naqvi, Rajesh Sankaran, Kishore Kunal, Praveen Elagala  | Broadcom initial draft  |
+| 1.0  | May-2024   | Syed Hasan Naqvi (Broadcom), Patrice Brissette (Cisco), Praveen Elagala (Broadcom), Rajesh Sankaran (Broadcom), Mike Mallin (Cisco), Kishore Kunal (Broadcom), Naveen Gamini (Cisco), Tapraj Singh (Cisco)   | Merged version   |
+| 1.1  | Jul-2024   | Syed Hasan Naqvi (Broadcom) | Updated kernel, SAI, config, and design sections |
+| 1.2  | Jul-2024   | Patrice Brissette (Cisco)   | Adding ShlOrchagent definition, addressing PR comments |
+| 1.3  | Aug-2024   | Patrice Brissette (Cisco)   | Updated SAI examples |
+
+
+<a id="About-this-Manual"></a>
+# About this Manual
+This document provides general information of EVPN VxLAN Multihoming feature implementation in SONiC.
+
+<a id="Scope"></a>
+# Scope
+This document describes the high level design and changes required for supporting Multihoming in EVPN VxLAN networks.
+This feature will provide [RFC 7432](https://datatracker.ietf.org/doc/html/rfc7432) and [RFC 8365](https://datatracker.ietf.org/doc/html/rfc8365) based Multihoming solution in SONiC.
+The ICCPd based MCLAG continue to co-exist as an alternate solution.
+
+<a id="Definition/Abbreviation"></a>
+# Definition/Abbreviation
+
+## Table 1: Abbreviations
+
+| Term     | Meaning                                   |
+| -------- | ----------------------------------------- |
+| (N)DF    | (Non-)Designated Forwarder                |
+| IRB      | Integrated Routing and Bridging           |
+| LVTEP    | Logical VTEP                              |
```

Comment:
can we remove abbreviation of LVTEP as LVTEP terminology is not being used throghout this HLD.

Reply:
@skumar041 , will be removed in next commit.